### PR TITLE
Add 2 by4 decoder

### PR DESCRIPTION
**Pull request by daumie**
simulation output of a 2by4 decoder plus accompanying code

### CODE REVIEW

File: dominic_monyoncho/queue/main.c [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#include <stdio.h>//input/output
```
Comment (Muriukidavid) [inline dominic_monyoncho/queue/main.c:1]: Look at the description of the commit message in this commit :-(

File: dominic_monyoncho/freescale/led_blinker.h [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+ *    *  Created on: August 1 2016
+ *     *      Author: https://github.com/daumie
+ *      */
+
+#ifndef LED_BLINKER_H_
+#define LED_BLINKER_H_
+#include <stdio.h>
+
+void  led_blinker(int byte1){
+            //int  E1, E2, E3, E4;
```
Comment (Muriukidavid) [inline dominic_monyoncho/freescale/led_blinker.h:13]: what's going on here?

File: dominic_monyoncho/queue/main.c (hunk 1 above)
Comment (daumie) [reply dominic_monyoncho/queue/main.c:1]: After second thought, the commit message is horrible and must have been a product of uncalled for haste.Any feedback improved is most welcomed

**Comment by daumie** [comment]
This wasn't part of the 2 by 4 decoder

**Pull request merged**